flink-table-store-hive/flink-table-store-hive-catalog/src/test/java/org/apache/flink/table/store/hive/HiveCatalogITCase.java

```diff
@@ -317,6 +319,29 @@ public void testHiveLock() throws InterruptedException {
         assertThat(count.get()).isEqualTo(100);
     }
 
+    @Test
+    public void testHiveTableLowerCase() throws Exception {
+        // create table
+        tEnv.executeSql("CREATE TABLE T ( A INT, b STRING ) WITH ( 'file.format' = 'avro' )")
```

Comment:
Actually, Flink SQL is case insensitive for table name too. We can add two cases:
```
    @Test
    public void testTableNameCaseInsensitive() throws Exception {
        // create table
        tEnv.executeSql("CREATE TABLE T ( a INT, b STRING ) WITH ( 'file.format' = 'avro' )")
                .await();
        tEnv.executeSql("SELECT a, b FROM T").print();
        tEnv.executeSql("SELECT a, b FROM t").print();
        assertThat(hiveShell.executeQuery("SELECT * FROM t").size()).isEqualTo(0);
        assertThat(hiveShell.executeQuery("SELECT * FROM T").size()).isEqualTo(0);
    }
```